redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/__tests__/incidentImageCarousel.test.js

```diff
@@ -17,6 +42,50 @@ describe('c-incident-image-carousel', () => {
         document.body.appendChild(element);
 
         const labelElement = element.shadowRoot.querySelector('p');
-        expect(labelElement.textContent).toContain('no images');
+        expect(labelElement.textContent).toContain(
+            'There are currently no images of damage for this case.'
+        );
+    });
+
+    it('renders two pictures when the wire mock returns two', () => {
+        // Create initial element
+        const element = createElement('c-incident-image-carousel', {
+            is: IncidentImageCarousel
+        });
+        document.body.appendChild(element);
+
+        // Emit data from @wire
+        getRelatedPicturesAdapter.emit(mockTwoImages);
+
+        return Promise.resolve().then(() => {
+            const imageEls = element.shadowRoot.querySelectorAll(
+                'lightning-carousel-image'
+            );
+            expect(imageEls.length).toEqual(mockTwoImages.length);
+            imageEls.forEach(function (value, index) {
+                expect.stringMatching(
+                    imageEls[index].src,
+                    /mockTwoImages[index].Id$/
```

Comment:
Creative solution with the inner function! What this should check is an exact check, not if only the `Id` value is passed. As in we expect to be `src` like `'/sfc/servlet.shepherd/version/download/WhatEverId`, and not `/mydownloads/WhatEverId`.

Reply:
It's specifically testing that the Id is the last string of characters in the URL. Since the full URL is hard coded by the component there's no reason to test this. Only test what can change.